src/js/components/LeftPane.js

```diff
@@ -43,42 +45,73 @@ const SectionContents = styled.div`
 const SectionHeader = styled.div`
   display: flex;
   flex-direction: row;
-  background-color: ${(props) => props.theme.colors.coconut};
+  background-color: var(--coconut);
```

Comment:
I do like how this simplifies how we select the color and gets rid of all the darn `${(props) => props.theme.finallySelectTheDamnColor}` boiler plate!

On the other hand, styled-components seems to really embrace defining css within js and this would mix the two paradigms a little. What if we just export the whole theme object and then import as needed? I also like being able to easily click into variables, which I don't think would work with css variables

Another thought (which doesn't yet apply to us, contradicts my last comment, but is worth thinking about for the future), is that the functionality that themes are supposed to provide is a restriction of sorts. So that certain components which inherit a given theme, can only use what's provided in that theme, while other components inherit different themes. Currently everything is already, and has been, global for us but it's just something to consider.

Reply:
In addition to the reasons you found, I'm really into css variables for a few reasons:
* They are accessible through javascript and stylesheets. This makes them an excellent bridge between total "css in js" and the standard stylesheet approach. Since we will have both patterns for a while, we can store the values of these variables in one place and use them in both contexts.
* They have live bindings, meaning that changing the value of --coconut in javascript will immediately update the color on the whole page. This makes "themeing" the app really nice. It we wanted to support several "skins" for brim, we could store vars like --bg-color, --text-color, --font-size. The change them all in Javascript and the whole page will update immediatly.
* They have excellent chrome devtools support

<img width="163" alt="Screen Shot 2020-08-04 at 1 02 16 PM" src="https://user-images.githubusercontent.com/3460638/89339269-baed5600-d652-11ea-9833-67b2893db2be.png">


I think we could use the theme for things with more than one property, and use css vars with things that are only shared values.

Does that make sense?